src/js/player.js

```diff
@@ -138,6 +138,9 @@ class Player extends Component {
     this.userActive_ = true;
     this.reportUserActivity();
     this.listenForUserActivity();
+
+    this.on('fullscreenchange', this.onFullscreenChange);
+    this.on('stageclick', this.onStageClick);
```

Comment:
`stageclick` is Flash specific, so we should just capture that in the flash tech and fire a click.